tools/megahit_contig2fastg/megahit_contig2fastg.xml

```diff
@@ -14,11 +14,11 @@
         '${contigs}' > '${fastg}'
     ]]></command>
     <inputs>
-        <param name="contigs" type="data" format="fastq,fastqsanger,fasta,fastq.gz,fastqsanger.gz" multiple="true" label="Single-end file(s)" help="FASTQ/FASTA/FASTQ.GZ files accepted" />
-        <param name="kmer" type="integer" value="99" label="K-mer length" />
+        <param name="contigs" type="data" format="fasta" label="Contig file" help="Ex: k99.contigs.fa" />
+        <param name="kmer" type="integer" value="99" label="K-mer length" help="Usually indicated within the filename (when derived from MEGAHIT)"/>
```

Comment:
I'm not sure this help is useful either as the filename is not seen by the user.

Reply:
Okay. Removed this as well.